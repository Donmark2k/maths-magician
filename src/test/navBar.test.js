import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from '../components/navBar';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <NavBar />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
