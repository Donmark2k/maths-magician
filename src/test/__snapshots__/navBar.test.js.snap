// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <h1>
     Maths Magicians
  </h1>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      |
    </li>
    <li>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      |
    </li>
    <li>
      <a
        href="/quotes"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
